Web/Edubase.Web.UI/Views/Tools/MergersTool.cshtml

```diff
@@ -260,8 +260,8 @@
                         <div class="govuk-date-input">
                             <div class="govuk-date-input__item">
                                 <div class="govuk-form-group">
-                                    <label for="amalgamationdate-day" class="govuk-label govuk-date-input__label">Day</label>
-                                    <input class="govuk-input date-text-day" type="text" v-model="mergeDateDay" id="amalgamationdate-day">
+                                    <label for="amalgamation-date-day" class="govuk-label govuk-date-input__label">Day</label>
```

Comment:
@JaswiniP should the for be changed here as we are only changing ids? FYI @RogerHowellDfE

Reply:
The `for` attribute of a `<label>` needs to match the `id` value of the `<input>` element the label is describing - updating one needs the other to be updated also, so updating both here is fine :)